src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -139,4 +181,19 @@ export const _Chatbox: React.FC<ChatboxProps> = ({
   );
 };
 
-export const Chatbox = React.memo(_Chatbox, isEqual);
+/**
+ * Resolves a bug in 'react-infinite-scroll-component' when
+ * 'next' function is not being called when height of initially loaded items is less than container's height.
+ * https://github.com/ankeetmaini/react-infinite-scroll-component/issues/217
+ */
+export function useTriggerScrollFix(messages: WithId<MessageToDisplay>[]) {
+  const ref = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (ref.current) ref.current.dispatchEvent(new CustomEvent("scroll"));
+  }, [messages.length, ref]);
+
+  return ref;
+}
```

Comment:
move to a hook

Reply:
But this is only relevant to this component. I thought that hooks in `/hooks` folder must serve some broad purpose and not just some specific component. Or do you mean just to extract it to a separate file in the Chatbox folder?

Reply:
If it's only used in this place, then yes, just extract into a separate file within the scope of Chatbox

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1922/commits/43b7b77a12a15f3a0fa0ccb7831da6e6648aa023